### PR TITLE
[ADD] sale_default_uom

### PR DESCRIPTION
**Pull request by AshishHirapara**
Added a module to select a different default UoM in the sales order line from the product template

### CODE REVIEW

**Review by francesco-ooops — APPROVED** [review]
Functional review ok!

**Review by SicurSam — APPROVED** [review]
LGTM

File: sale_default_uom/models/product_template.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+from odoo import api, fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    sale_uom_id = fields.Many2one("uom.uom", "Default UoM for Sales")
```
Comment (rafaelbn) [inline sale_default_uom/models/product_template.py]:
```suggestion
    sale_uom_id = fields.Many2one("uom.uom", "Default Sale UoM")
```

File: sale_default_uom/readme/USAGE.rst [hunk 2]
```diff
@@ -0,0 +1,6 @@
+To use features of this module, you need to:
+
+#. Go to Sales > Products > Products
+#. Open a product.
+#. You will see a field called 'Default UOM for sales' on the form view.
```
Comment (rafaelbn) [inline sale_default_uom/readme/USAGE.rst]:
```suggestion
#. You will see a field called 'Default Sale UoM' on the form view.
```

**Review by rafaelbn — APPROVED** [review]
Just UX remarks with coherence of other fields and shorter

Unit of Measure 
Purchase Unit of Measure
Default Sale UoM

**Comment by rafaelbn** [comment]
Dear @AshishHirapara could you please review my suggestions please? You don't have to accept it just review it 😄 thank you!

File: sale_default_uom/models/product_template.py (hunk 1 above)
Comment (AshishHirapara) [reply sale_default_uom/models/product_template.py]: I will update it

File: sale_default_uom/readme/USAGE.rst (hunk 2 above)
Comment (AshishHirapara) [reply sale_default_uom/readme/USAGE.rst]: I will update it

**Comment by AshishHirapara** [comment]
Dear @rafaelbn I have reviewed and updated the changes you have mentioned in the comment regarding field names. But the code coverage is failing due to another module. Can you tell me what I can do here?

**Pull request merged**